x-pack/plugin/security/qa/multi-cluster/src/javaRestTest/java/org/elasticsearch/xpack/remotecluster/RemoteClusterSecuritySpecialUserIT.java

```diff
@@ -33,10 +35,11 @@
 import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.Matchers.greaterThanOrEqualTo;
 
-@LuceneTestCase.AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/94939")
 public class RemoteClusterSecuritySpecialUserIT extends AbstractRemoteClusterSecurityTestCase {
 
     private static final AtomicReference<Map<String, Object>> API_KEY_MAP_REF = new AtomicReference<>();
+    private static final AtomicReference<Tuple<LocalClusterConfigProvider, LocalClusterConfigProvider>> CLIENT_AUTH_CONFIG_PROVIDERS =
```

Comment:
Nit: a record `LocalClusterConfigProviders(fcProvider, qcProvider)` would make this a bit easier to parse